toolkits/nature/packages/nature-hero/README.md

```diff
@@ -42,7 +59,7 @@ You can see an example in the [Hero's `demo` folder](https://github.com/springer
 
 ## Help improve this page
 
-If you’ve got a question, idea or suggestion about how to improve this component or guidance, post in the [#design-systems Slack channel](https://springernature.slack.com/archives/C75DHBTBP).
+If you’ve got a question, idea or suggestion about how to improve this component or guidance, post in the [#ask-elements Slack channel](https://springernature.slack.com/archives/CNBTFLBLP).
```

Comment:
I've added the url needed

Reply:
I really thought I had changed that already. Thanks.